.travis.yml

```diff
@@ -30,7 +30,7 @@ after_deploy:
   - >
     if [[ "$TRAVIS_BRANCH" == "master" ]]; then
       echo "Updating import map to point to new version of @openmrs/esm-patient-chart-widgets"
-      statuscode=$(curl --output /dev/null --write-out %{http_code} -u $DEPLOYER_USERNAME:$DEPLOYER_PASSWORD -d '{ "service":"@openmrs/esm-patient-chart-widgets","url":"https://spa-modules.nyc3.digitaloceanspaces.com/@openmrs/esm-patient-chart-widgets/'$TRAVIS_COMMIT'/openmrs-esm-patient-chart-widgets.js" }' -X PATCH $DEPLOYER_HOST/services\?env=prod -H "Accept:application/json" -H "Content-Type:application/json")
+      statuscode=$(curl --output /dev/null --write-out %{http_code} -u $DEPLOYER_USERNAME:$DEPLOYER_PASSWORD -d '{ "service":"@openmrs/esm-patient-chart-widgets-app","url":"https://spa-modules.nyc3.digitaloceanspaces.com/@openmrs/esm-patient-chart-widgets/'$TRAVIS_COMMIT'/openmrs-esm-patient-chart-widgets.js" }' -X PATCH $DEPLOYER_HOST/services\?env=prod -H "Accept:application/json" -H "Content-Type:application/json")
```

Comment:
Why? Can we leave it without the `-app`? Its just a collection of extensions.

Reply:
This is a mistake. I'd made this change after my conversation with you on slack but improvements were later made that made this change unnecessary. I just forgot to undo it. Will fix now.